common/extract_repo.py

```diff
@@ -0,0 +1,189 @@
+import sys
```

Comment:
add copyright
run pylint and add comments

Reply:
done